Tests/Marketplace/marketplace_services.py

```diff
@@ -1561,36 +1561,21 @@ def format_metadata(self, user_metadata, pack_content_items, integration_images,
         finally:
             return task_status
 
-    def _get_changelog(self, index_folder_path):
-        """ Gets the pack changelog data.
-        Args:
-            index_folder_path (str): downloaded index folder directory path.
-        Returns:
-            dict: Get the changelog from downloaded index
-        """
-        changelog_index_path = os.path.join(index_folder_path, self._pack_name, Pack.CHANGELOG_JSON)
-        changelog = {}
-        if os.path.exists(changelog_index_path):
-            with open(changelog_index_path, "r") as changelog_file:
-                changelog = json.load(changelog_file)
-        return changelog
-
     def _get_pack_creation_date(self, index_folder_path):
         """ Gets the pack created date.
         Args:
             index_folder_path (str): downloaded index folder directory path.
         Returns:
             datetime: Pack created date.
         """
-        earliest_changelog_released_date = datetime.utcnow().strftime(Metadata.DATE_FORMAT)
-        changelog = self._get_changelog(index_folder_path)
+        created_time = datetime.utcnow().strftime(Metadata.DATE_FORMAT)
+        metadata = load_json(os.path.join(index_folder_path, self._pack_name, Pack.METADATA))
```

Comment:
If the file dose not exist, I think this can create exception. no?

Reply:
we check it in `load_json`